### ISSUE TITLE
Resume fails

### ISSUE THREAD

**Issue by mfseeker** [UniversalMediaServer/UniversalMediaServer#1410]
I have never been able to get Resume to work. It fails in a variety of ways. This session was typical. I selected the "Resume: Remains of the Day" entry. The screen displayed "Please Wait..." for the better part of a minute. Then it reverted to the menu. This is a best case example. More frequently, attempts to use a resume file result in crashing the Moxi Headed Gateway set top renderer.
[ums_dbg.zip](https://github.com/UniversalMediaServer/UniversalMediaServer/files/1424434/ums_dbg.zip)

**Comment by Sami32** [comment]
Yes, sadly the "Resume" code is flawed, and many issues reports have been done. The original developper of this code is not around anymore.
It will need a whole refactoring, but many other thing need to be addressed before, IMO.

**Comment by SpitFire-666** [comment]
Delete my comment if it's been covered already... Resume has never worked for me on UMS in a web browser. It just plays the video from the start again without any thought or errors.
